md/applications.md

```diff
@@ -44,7 +44,7 @@ To integrate a process into an application, you specify a link in an application
 ## Defining a new application
 
 ::: info
-Since Bonita BPM 7.5.0, the definition of applications can be done from the studio, which makes it more straightforward to develop and package with other application artifacts.
+Since Bonita 7.5.0, the definition of applications can be done from the studio, which makes it more straightforward to develop and package with other application artifacts.
```

Comment:
version 7.5.0 was still named Bonita BPM, so I think it should be kept as is...
to be discussed.